docker/entry.sh

```diff
@@ -42,33 +42,11 @@ if [ -z "$GEOIPUPDATE_ACCOUNT_ID" ] || [ -z  "$GEOIPUPDATE_LICENSE_KEY" ] || [ -
     exit 1
 fi
 
-# Create configuration file
+# Create an empty configuration file. All configuration is provided via
+# environment variables or command line options, but geoipupdate still
+# expects a configuration file to exist.
 echo "# STATE: Creating configuration file at $conf_file"
```

Comment:
Maybe we could drop this line then. It might just be confusing.

We should update docker.md as well regarding the config file.